.github/workflows/pact.yml

```diff
@@ -0,0 +1,63 @@
+name: Pact Contract Tests
+on: [push]
+jobs:
+  tests:
+    name: Test
+    runs-on: ubuntu-16.04
+    env:
+      BUNDLE_ENTERPRISE__CONTRIBSYS__COM: ${{ secrets.BUNDLE_ENTERPRISE__CONTRIBSYS__COM }}
+      DATABASE_URL: "postgis://vets-api:postgres@127.0.0.1:5432/vets-api-test"
+      RAILS_ENV: test
+      PGHOST: localhost
+      PGUSER: vets-api
+      PGPASSWORD: postgres
+      "Settings.test_database_url": "postgis://vets-api:postgres@127.0.0.1:5432/vets-api-test"
+      "Settings.saml.cert_path": "spec/support/certificates/ruby-saml.crt"
+      "Settings.saml.key_path": "spec/support/certificates/ruby-saml.key"
+      "Settings.binaries.clamdscan": "clamscan"
+
+    services:
+      postgres:
+        image: postgis/postgis:11-2.5
+        env:
+          POSTGRES_USER: vets-api
+          POSTGRES_DB: vets-api-test
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 15
+
+      redis:
+        image: circleci/redis:5.0-alpine
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: "us-gov-west-1"
+
+      - name: Get bot token from Parameter Store
+        uses: marvinpinto/action-inject-ssm-secrets@v1.1.1
+        with:
+          ssm_parameter: /dsva-vagov/pact-broker/utility/PACT_BROKER_BASIC_AUTH_PASSWORD
+          env_variable_name: PACT_BROKER_BASIC_AUTH_PASSWORD
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Install dependencies (pdftk, poppler, imagemagick)
+        run: sudo apt install imagemagick pdftk poppler-utils
+
+      - name: Setup Database
+        run: bundle exec rake db:schema:load
+
+      - name: Verify stable Pacts
+        run: bundle exec rake pact:verify
+        env:
+          PACT_BROKER_BASIC_AUTH_USERNAME: pact-broker
```

Comment:
Does the PACT_BROKER_BASIC_AUTH_PASSWORD need to be set here too? Or maybe line 47 takes care of that

Reply:
Yeah, the `Get bot token from Parameter Store` step handles that, so it's not needed here.